scripts/download/wsl.ts

```diff
@@ -6,9 +6,9 @@ import fs from 'fs';
 import os from 'os';
 import path from 'path';
 
-import { download } from '../lib/download.mjs';
+import { download } from '../lib/download';
 
-export default async function main() {
+export default async function main(): Promise<void> {
   const v = '0.26';
```

Comment:
This should be reading from the yaml?

Reply:
Yes it should. But I figured I'd avoid making many more changes so this PR didn't get too big. I've actually already done this in the branch that will become the next PR. Is it alright to leave it here?